src/pages/news/2020-05-23-cach_kiem_tien_an_toan_voi_bitcoin.vn.md

```diff
@@ -0,0 +1,57 @@
+
+### Hỗ trợ hoàn toàn bởi BTC
+Không giống như nhiều token tổng hợp trên thị trường, TBTC được Bitcoin hỗ trợ hoàn toàn với tỷ lệ 1: 1. Điều này có nghĩa là đối với mỗi TBTC đang lưu hành, luôn có ít nhất một BTC giá trị bị khóa trong hệ thống. Người nắm giữ token tự tin rằng họ không có nguy cơ trở thành nạn nhân mất tài sản. Hơn nữa, vì giá trị có thể bị thay đổi trên BTC, TBTC và ETH, BTC có thể bị mất cắp sẽ được bồi thường nhiều hơn bởi ETH tịch thu của người ký phạm tội. 
+
+![tbtc2.png](https://images.hive.blog/DQmTicqubo2t4KK8C1sc1JaLvp1xmVxVPqtABkDQbAefLVG/tbtc2.png)
+
+
+### Sử dụng Bitcoin để tận dụng sức mạnh của Ethereum
+
+Đúng là Bitcoin thống trị rõ ràng về mặt vốn hóa thị trường tiền điện tử tổng thể, với giá trị kết hợp lớn hơn tất cả các token khác cộng lại. Nhưng Ether Ethereum đã thu hút được cộng đồng nhà phát triển lớn nhất. Điều này có nghĩa là nguồn vốn tiền điện tử lớn nhất đã bị cắt đứt một cách hiệu quả từ nhiều dự án, làm mở rộng những gì có thể với chuỗi khối và hứa hẹn viết lại cái nhìn của thế giới đối với lĩnh vực tài chính và các lĩnh vực quan trọng khác.
+
+Giờ đây, tBTC kết hợp tính ổn định và tính thanh khoản của Bitcoin với sự đổi mới và động lực phát triển của Ethereum. Khi token khởi chạy trên mạng chính Ethereum, nó sẽ cung cấp cho mọi người một cách minh bạch, phi tập trung và an toàn để đưa BTC hoạt động. Người dùng tBTC sẽ có thể sử dụng Bitcoin của họ để tham gia vào các dự án và cơ hội sáng tạo đang được phát triển bởi cộng đồng đam mê Ethereum.
+
+### Theo tBTC
+
+Bitcoin chiếm phần lớn giá trị tài sản điện tử trên thế giới, và giờ đây, có một cách an toàn, phi tập trung, minh bạch để BTC đó được đưa vào hoạt động để hỗ trợ các trường hợp sử dụng DeFi đầy hứa hẹn và các dự án khác trên Ethereum. Nhóm của chúng tôi đã xác định một số trường hợp sử dụng DeFi cho tBTC nói riêng và sẽ sớm chia sẻ thông tin và thông báo bổ sung. Cho đến ngày hôm nay, tBTC vẫn [hoạt động trên mạng thử nghiệm Ropsten Ethereum](https://tbtc.network/news/2020-02-14-ropsten). 
+
+Tham gia danh sách nhận thư tBTC [để cập nhật](https://tbtc.network/#mailing-list), bao gồm thông tin về việc ra 
+mắt sắp tới của tBTC trên mạng chính Ethereum. Để tìm hiểu thêm về thiết kế kỹ thuật tBTC, hãy đọc [thông số kỹ thuật](http://docs.keep.network/tbtc/index.pdf). Tham gia kênh [Keep #tbtc trên Discord](https://discord.gg/wYezN7v) để biết các câu hỏi kỹ thuật về tBTC và [tbtc.js](https://tbtc.network/news/2020-02-14-announcing-tbtc-js) và theo dõi các Twitter này để biết tin tức và cơ hội tham gia:
+
```

Comment:
We've started switching these weird bits out for a more natural end in other posts- can you pull that instead?

Reply:
I updated all posts.